src/tests/daos_perf.c

```diff
@@ -1110,6 +1115,12 @@ main(int argc, char **argv)
 		}
 	}
 
+	if (ts_verify_fetch && ts_shuffle) {
```

Comment:
Don't we shuffle the whole array?   Not sure why this isn't possible in this case

Reply:
the verify flag was not working anyway before since shuffle was always on. i just catch it now and quit.
The array is shuffled after every phase (update, fetch, verify), so shuffling and verify would verify at the wrong offset and verification will fail. yes this can be fixed to work, but I just did the minimum here :-)